pennylane/devices/default_tensor.py

```diff
@@ -199,7 +199,9 @@ class DefaultTensor(Device):
     The backend uses the ``quimb`` library to perform the tensor network operations, and different methods can be used to simulate the quantum circuit.
     The supported methods are Matrix Product State (MPS) and Tensor Network (TN).
 
-    This device does not currently support finite-shots or differentiation. At present, the supported measurement types are expectation values, variances and state measurements.
+    This device does not currently support finite-shots or differentiation with diff_method="backprop", "adjoint", or "device". Other differentiation methods such as 
```

Comment:
We may want to link to the list of `other differentiation methods`.
```suggestion
    This device does not currently support finite-shots or differentiation with diff_method="backprop", "adjoint", or "device". `Other differentiation methods <https://docs.pennylane.ai/en/stable/code/qml_gradients.html>`_ such as 
```